Fixes a handful of plumbing issues on Birdshot station

Derpguy3 opened the pull request:
## About The Pull Request
There's three sinks in question for this one. A sink in the bar, a sink in medical's locker/storage room and one in the perma-brig's kitchen.

Additionally adds a water tank connection to the shower room by dorms, as the plumbing for it existed, but there was no source. (Removed some redundant piping, as well as moving a rack to make space.)

All the sinks have a plumbed connection leading to a water tank, but none of the tanks were actually facing towards the pipes, making none of them supply water in the first place.

This PR rotates the water tanks, and adds one tank so that they connect with the pipes and actually supply water.
## Why It's Good For The Game
A plumbing system works much better when connected to the water source.
## Changelog
:cl:
fix: Three stationary water tanks on Birdshot which supplied a sink in the bar, a sink in the medical storage and a sink in the perma-brig's kitchen are now properly rotated and connected with the plumbing.
fix: The unisex showers on Birdshot have been given a water tank for its plumbing.
/:cl:

Derpguy3 committed:
Rotates two stationary water tanks

Rotates the direction of two water tanks on birdshot;
- A water tank connected to the bar's sink.
- A water tank connected to a sink in medical's locker/storage room.

AdipemDragon commented:
is it too late to ask to do the perma mess hall one too? Since that one's also just a disconnected mess

Derpguy3 committed:
Fixes two more plumbing issues

- Rotates a stationary water tank in the perma-wing canteen
- Adds a stationary water tank in nearby maintenance to supply the shower room. (Also removes some redundant plumbing to line up with the tank, and the position of a rack to place the tank.)

Derpguy3 commented:
> is it too late to ask to do the perma mess hall one too? Since that one's also just a disconnected mess

Thanks for catching that one! I'd forgotten to check perma when I had been looking around.

Derpguy3 committed:
Fixes my mistakes

I plumbed it wrong. Connection pipes don't rotate for corner pieces.
